### PR TITLE
Scatter 6

### LINKED ISSUE #202: scatter_6
Scatter plot on a polar projection.

script: https://www.ncl.ucar.edu/Applications/Scripts/scatter_6.ncl
plot: https://www.ncl.ucar.edu/Applications/Images/scatter_6_lg.png

### PR DESCRIPTION
**Pull request by clairefio**
Closes #202:

scatter_6

### CODE REVIEW

File: Plots/Scatter/NCL_scatter_6.py [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+###############################################################################
+# Import packages:
+import numpy as np
+import matplotlib.pyplot as plt
+
+###############################################################################
+# Create dummy data:
+
+numpoints = 100
+lon = np.random.randint(0, 360, numpoints)
```
Comment (hCraker) [inline Plots/Scatter/NCL_scatter_6.py:26]: `randint` is a totally valid way of generating random data. I did notice that the NCL script uses a random uniform function. If you'd like a more uniform distribution, there is a [function](https://het.as.utexas.edu/HET/Software/Numpy/reference/generated/numpy.random.uniform.html) in `numpy` for that. It's up to you which you'd like to use.

File: Plots/Scatter/NCL_scatter_6.py [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+               lat[x],
+               color=colors[x % 10],
+               s=bins[x % 10],
+               edgecolors='black',
+               linewidths=1,
+               alpha=0.9,
+               zorder=2)
+
+# set the labels and locations of the angular gridlines
+linelabels = ('O', '30E', '60E', '90E', '120E', '150E',
```
Comment (hCraker) [inline Plots/Scatter/NCL_scatter_6.py:61]:
```suggestion
linelabels = ('0', '30E', '60E', '90E', '120E', '150E',
```

File: Plots/Scatter/NCL_scatter_6.py [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+
+# Create array of marker colors
+colors = ("limegreen", "orange", "green", "red", "yellow", "purple",
+          "blue", "red", "brown", "crimson", "skyblue")
+
+# Create array of marker sizes
+bins = np.linspace(100, 2000, 10)
+
+# Plot all points
+# longitude points must be transformed to be plotted on polar projection
```
Comment (hCraker) [inline Plots/Scatter/NCL_scatter_6.py:49]: Can you elaborate on this transformation? You're going from degrees to radians, but make sure you specify that in the comments.

**Comment by clairefio** [comment]
Updated plot:

scatter_6

**Pull request merged**